pkg/blockdeviceclaim/v1alpha1/build.go

```diff
@@ -253,25 +254,50 @@ func (b *Builder) WithCapacity(capacity string) *Builder {
 
 // WithOwnerReference sets the OwnerReference field in BDC with required
 //fields
-func (b *Builder) WithOwnerReference(spc *apis.StoragePoolClaim) *Builder {
-	if spc == nil {
+// OwnerReference to PVC is for local-device provisioner
+func (b *Builder) WithOwnerReference(owner interface{}) *Builder {
```

Comment:
Instead of using empty interface, use [this](https://github.com/kubernetes/apimachinery/blob/master/pkg/apis/meta/v1/types.go#L306) `OwnerReference` struct here.